bin/yugabyted

```diff
@@ -1288,6 +1305,12 @@ class ControlScript(object):
             cur_parser.add_argument(
                 "--ui", choices=BOOL_CHOICES, default="false", metavar="BOOL",
                 help="Toggle enabling or disabling webserver UI. Default false.")
+            cur_parser.add_argument(
+                "--ysql_enable_auth", choices=BOOL_CHOICES, metavar="BOOL",
```

Comment:
add an argparse.SUPPRESS so that it does not show up in the default help list.

Reply:
okay. will update.